connectivity/drivers/lora/TARGET_STM32WL/radio_board_if.h

```diff
@@ -0,0 +1,238 @@
+  * the License. You may obtain a copy of the License at:
+  *                             www.st.com/SLA0044
+  *
+  ******************************************************************************
+  */
+
+/* Define to prevent recursive inclusion -------------------------------------*/
+#ifndef RADIO_BOARD_IF_H
+#define RADIO_BOARD_IF_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* DBG LCH: static definition: could be moved up in call stack (platform.h) */
+/* STM32WL BSP driver code generated by MX */
+#define MX_BOARD_PSEUDODRIVER 
+  
+  
+/* Includes ------------------------------------------------------------------*/
+#include "stm32wlxx_hal.h"
+  
+/* USER CODE BEGIN include */
+
+/* USER CODE END include */
+
+/* Exported defines ----------------------------------------------------------*/
+#ifdef MX_BOARD_PSEUDODRIVER
+#define MX_NUCLEO_WL55JC1
+#endif /* MX_BOARD_PSEUDODRIVER */
+#if defined(USE_BSP_DRIVER)
+/* code generated by MX does not support BSP */
+/* In order to use BSP driver, add the correspondent files in the IDE workspace */
+/* and define USE_BSP_DRIVER in the preprocessor definitions  or in platform.h */
+#define RBI_CONF_RFO_LP_HP  RADIO_CONF_RFO_LP_HP
+#define RBI_CONF_RFO_LP     RADIO_CONF_RFO_LP
+#define RBI_CONF_RFO_HP     RADIO_CONF_RFO_HP
+#elif defined(MX_NUCLEO_WL55JC1)
+#define RBI_CONF_RFO_LP_HP  0
+#define RBI_CONF_RFO_LP     1
+#define RBI_CONF_RFO_HP     2
+#else
+/* USER CODE BEGIN Board Definition */
+#error user to provide its board definitions
+/* USER CODE END Board Definition */
+#endif  /* USE_BSP_DRIVER  || MX_NUCLEO_WL55JC1  */
+
+#if defined(USE_BSP_DRIVER)
+/* code generated by MX does not support BSP */
+/* In order to use BSP driver, add the correspondent files in the IDE workspace */
+/* and define USE_BSP_DRIVER in the preprocessor definitions  or in platform.h */
+
+#elif defined(MX_NUCLEO_WL55JC1)
+
+/* Indicates the type of switch between the ones proposed by CONFIG Constants
+ */
+#define RBI_CONF_RFO                        RBI_CONF_RFO_LP_HP
+
+/* Radio maximum wakeup time (in ms) */
+#define RF_WAKEUP_TIME                     10U
+
+/* Indicates whether or not TCXO is supported by the board
+ * 0: TCXO not supported
+ * 1: TCXO supported
+ */
+#define IS_TCXO_SUPPORTED                   1U
+
+/* Indicates whether or not DCDC is supported by the board
+ * 0: DCDC not supported
+ * 1: DCDC supported
+ */
+#define IS_DCDC_SUPPORTED                   1U
+
+#else
+/* USER CODE BEGIN Exported Parameters */
+#error user to provide its board definitions
+/* USER CODE END Exported Parameters */
+#endif  /* USE_BSP_DRIVER  || MX_NUCLEO_WL55JC1  */
+
+#if defined(USE_BSP_DRIVER)
+/* code generated by MX does not support BSP */
+/* In order to use BSP driver, add the correspondent files in the IDE workspace */
+/* and define USE_BSP_DRIVER in the preprocessor definitions  or in platform.h */
+
+#elif defined(MX_NUCLEO_WL55JC1)
```

Comment:
@JeanMarcR  @jeromecoutant  We should make sure that those control pins are not offered for other preipherals / purpose in our mbed pinmap !

Reply:
file has been removed, pins are now defined in mbed_app.json